### PR TITLE
Song suggestion feature

### PR DESCRIPTION
**Pull request by unclesam79**

### CODE REVIEW

File: submission_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import random as r
+import shutil
+import requests
+
+import praw
+
+reddit = praw.Reddit(client_id='AmJbMF6Sh56JNg',
+                             client_secret='1K1ovK2ku9EP_usjLR-8CFRiQB8',
+                             user_agent='windows:test:v0.1 (by u/unclesam79)')
```
Comment (harshil21) [inline submission_fetcher.py]: Probably not a good idea to share this with the world. Add to .gitignore

Comment (unclesam79) [reply submission_fetcher.py]: > Probably not a good idea to share this with the world. Add to .gitignore

oh SHIT

File: submission_fetcher.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            for i in range(self.filter_count):
+                for submission in reddit.subreddit(sub).top(time_filter=self.filters[i][0], limit=self.filters[i][1]):
+                    if submission not in self.SUBMISSIONS:  # Unique submissions only
+                        self.SUBMISSIONS.append(submission)
+        print(f"Got {len(self.SUBMISSIONS)} submissions in total from {self.subreddits}.")
+
+    def get_post(self):
+        """Return a random submission from SUBMISSIONS"""
+
+        if self.SUBMISSIONS == []:
```
Comment (harshil21) [inline submission_fetcher.py]: Better to use `if not self.SUBMISSIONS` according to [PEP-8](https://www.python.org/dev/peps/pep-0008/#id51)

File: submission_fetcher.py (hunk 1 above)
Comment (unclesam79) [reply submission_fetcher.py]: Credentials have been revoked and new credentials have been shifted to a gitignored file.

**Review by harshil21 — APPROVED** [review]

File: submission_fetcher.py (hunk 2 above)
Comment (unclesam79) [reply submission_fetcher.py]: Done.

**Pull request merged**